### PR TITLE
Removed g728-enc5 test

### PR DESCRIPTION
**Pull request by ludomal**
Fixing one of the case in #53.

g728-enc5 test was not part of the original STL package and there is no reference vector for it.

### CODE REVIEW

File: src/g728/g728float/CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,10 @@
 file(GLOB G728FLOAT_SRC *.c)
 add_executable(g728 ${G728FLOAT_SRC})
+target_compile_definitions(g728 PUBLIC USEDOUBLES=1)
```
Comment (dennisguse) [inline src/g728/g728float/CMakeLists.txt]: What is the reason why this define is activated?
Was active in the STL2009?

File: src/g728/g728float/CMakeLists.txt [hunk 2]
```diff
@@ -15,9 +18,6 @@ add_test(g728-enc3-verify ${CMAKE_COMMAND} -E compare_files ../test_data/incw3.b
 add_test(g728-enc4 ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/g728 -little enc ../test_data/in4.bin ../test_data/in4.bin.float.out)
 add_test(g728-enc4-verify ${CMAKE_COMMAND} -E compare_files ../test_data/incw4.bin ../test_data/in4.bin.float.out)
 
-add_test(g728-enc5 ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/g728 -little enc ../test_data/in5.bin ../test_data/in5.bin.float.out)
```
Comment (dennisguse) [inline src/g728/g728float/CMakeLists.txt]: Should we then also remove the test file?

File: src/g728/g728float/CMakeLists.txt (hunk 1 above)
Comment (ludomal) [reply src/g728/g728float/CMakeLists.txt]: Yes, USEDOUBLES was active in STL2009 (see https://github.com/openitu/STL/blob/master/src/g728/g728float/makefile.unx).

File: src/g728/g728float/CMakeLists.txt (hunk 2 above)
Comment (ludomal) [reply src/g728/g728float/CMakeLists.txt]: ../test_data/in5.bin is used as test vector for the fixed point version so it should not be removed.

Comment (ludomal) [reply src/g728/g728float/CMakeLists.txt]: Could do... I do not know why this reference was missing originally. 
I will generate that file and update the pull request.

File: src/g728/g728float/CMakeLists.txt (hunk 1 above)
Comment (dennisguse) [reply src/g728/g728float/CMakeLists.txt]: I see.
Could you put this into a separate commit as it is actually a significant change?
Or put into the commit message?

**Comment by ludomal** [comment]
I have created the reference vector for g728-enc5 and restored that test in CMakeList.
I have also created a separate commit for USEDOUBLES.

**Comment by ludomal** [comment]
The commits should be clean up now. Thanks.

**Comment by othiman** [comment]
> The commits should be clean up now. Thanks.

Perfect. :-)

**Pull request merged**